doc/release/upcoming_changes/14255.improvement.rst

```diff
@@ -0,0 +1,4 @@
+`unique` has consistent axes order (except the chosen one) when `axis` is not None
+----------------------------------------------------------------------------------
+Using `moveaxis` instead of `swapaxes` when implementation, so that the ordering
+of axes except the axis in arguments will not be broken.
```

Comment:
The titile should include `numpy.unique` not just `unique

`axis`, `moveaxis`, `swapaxis` should have double-backticks so they are formatted as code, not internal links.

`"when implementation" -> "in `numpyunique`" (note single backticks around `numpy.unique`